packages/eslint-plugin-next/lib/index.js

```diff
@@ -33,5 +33,13 @@ module.exports = {
         '@next/next/no-duplicate-head': 2,
       },
     },
+    'core-web-vitals': {
+      plugins: ['@next/next'],
+      // extends: ['plugin:@next/next/recommended'],
```

Comment:
open question: should the config include only the overrides, or extend from the recommended config so it can be used standalone

Reply:
I would be in favor of extending the recommended config so it could be used on its own. That's the current behavior and I even have a PR to update the [docs](https://github.com/vercel/next.js/blob/cc17e9b89c6fd991ae7b470212cc534057ee58bb/docs/basic-features/eslint.md#:~:text=the%20first%20time-,.,-%7B%0A%20%20%22extends%22%3A%20%22next/core) and default setup to show how the config can be used standalone as well.